lib/rules/validate-jsdoc/enforce-existence.js

```diff
@@ -1,38 +1,91 @@
+var assert = require('assert');
+
 module.exports = enforceExistence;
 module.exports.scopes = ['function'];
 module.exports.options = {
-    enforceExistence: {allowedValues: [true, 'exceptExports']}
+    enforceExistence: true
+};
+
+/**
+ * @param {Object} options
+ */
+enforceExistence.configure = function(options) {
+
+    // set default policy
+    enforceExistence.policy = {
```

Comment:
@zxqfox How do you access this._policy from enforceExistence when it is defined as a property of enforceExistence.configure?

When using this._policy for both enforceExistence and enforceExistence.configure it fails tests.